admin/app/controllers/authenticated/certification-centers/new.js

```diff
@@ -0,0 +1,25 @@
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import Controller from '@ember/controller';
+
+export default class NewController extends Controller {
+
+  @service notifications;
+
+  @action
+  goBackToCertificationCentersList() {
+    this.transitionToRoute('authenticated.certification-centers');
+  }
+
+  @action
+  async addCertificationCenter(event) {
+    event.preventDefault();
+    try {
+      await this.model.save();
+      this.notifications.success('Le centre de certif a été créé avec succès.');
+      this.transitionToRoute('authenticated.certification-centers.list');
+    } catch (error) {
```

Comment:
Coté back je vois du JOI:

name: Joi.string()
    .required()
    .messages({
      'string.empty': 'Le nom n’est pas renseigné.',
    }),

  type: Joi.string()
    .required()
    .valid('SCO', 'SUP', 'PRO')
    .messages({
      'string.empty': 'Le type n’est pas renseigné.',
      'any.only': 'Le type du centre de certification doit avoir l’une des valeurs suivantes: SCO, SUP, PRO.',
    }),

  externalId: Joi.string()
    .min(0)
    .max(255)
    .messages({
      'string.max': 'L‘externalId doit faire maximum 255 caractères.'
    })

Prévoir d'afficher les messages des bad request dans le catch même s'il a des validateurs coté front et un message par défault pour les erreurs irrécouvrable 500:

```suggestion
const errorDetail = get(response, 'errors[0].detail', 'Une erreur interne est survenue, nos équipes sont en train de résoudre le problème. Veuillez réessayer ultérieurement.');
this.notifications.error(errorDetail);
```

Reply:
Les messages n'étaient pas bien remontés mais c'est bon maintenant, les messages sont bien en français. Concernant, la gestion d'une erreur plus globale, on affiche déjà "Une erreur est survenue." dans tous les cas et vu que Pix Admin s'adresse à des utilisateurs internes je ne suis pas sûr que l'on ait besoin d'être plus précis.